packages/google_mlkit_document_scanner/android/src/main/java/com/google_mlkit_document_scanner/GoogleMlkitDocumentScannerPlugin.java

```diff
@@ -0,0 +1,238 @@
+package com.google_mlkit_document_scanner;
```

Comment:
Follow the same design we have for the other plugins. You should have 2 files: `DocumentScanner.java` and `GoogleMlKitDocumentScannerPlugin.java`

Reply:
Yeah! About that.
Separating the file won't work. 
I did that initially, but we need to have the FlutterPlugin and ActivityAware on the same abstract class. I spent sometime on it. thank for @ablause for bailing my out on it.